build.rs

```diff
@@ -0,0 +1,14 @@
+#[cfg(windows)]
+extern crate winres;
+
+#[cfg(windows)]
+fn main() {
+    if std::env::var("PROFILE").unwrap() == "release" {
+        let mut res = winres::WindowsResource::new();
+        res.set_icon("assets/favicon.ico");
+        res.compile().expect("Unable to setup windows resources informations.");
+    }
+}
+
+#[cfg(not(target_os = "windows"))]
+fn main() {}
```

Comment:
No linux build as of now ?

Reply:
No, this file is only needed for binary icon and information on windows. 31b22e4

https://github.com/lucid-kv/lucid/pull/7#issuecomment-549103441

Reply:
It's not really important but I like details.